guides/stable/connecting_events.md

```diff
@@ -1,6 +1,6 @@
 (connecting-events)=
 
-## Hooking up your own events
+# hooking up your own events
```

Comment:
I personally would like to see us use sentence case for titles (eg: starting with a capital letter). This hasn't historically been the case, I'm not really sure why.

Reply:
should this go with plugin section?

Reply:
> should this go with plugin section?

While this is commonly done in plugins, you can definitely hook up events in other workflows so I think this should live outside the plugin section.

Reply:
Maybe we need the concept of tags. 
There is a lot of content that we want, say in the plugin section, but also belongs somewhere else *as well*. If we can make these accessible from both places that might help

Reply:
Yes, I do think tags would work great here as we don't have a huge amount of information yet and could create them rather easily.